pr_insight/cli_pip.py

```
@@ -6,18 +6,19 @@ def main():
     # Fill in the following values
     provider = "github"  # GitHub provider
     user_token = "..."  # GitHub user token
-    openai_key = "..."  # OpenAI key
-    pr_url = "..."  # PR URL, for example 'https://github.com/Khulnasoft/pr-insight/pull/809'
-    command = "/review"  # Command to run (e.g. '/review', '/describe', '/ask="What is the purpose of this PR?"')
+    openai_key = "ghs_afsdfasdfsdf"  # Example OpenAI key
+    pr_url = "..."  # PR URL, for example 'https://github.com/KhulnaSoft/pr-insight/pull/809'
+    command = "/improve"  # Command to run (e.g. '/review', '/describe', 'improve', '/ask="What is the purpose of this PR?"')
```

Ruff (0.8.2)

Line too long (125 > 120)

(E501)